templates/resources/network_policy.md.tmpl

```diff
@@ -0,0 +1,184 @@
+---
+Title: "Network Policy Resource"
+Description: |-
+    Creating the Tanzu Kubernetes network policy resource.
+---
+
+# Managing Network Communication for Your Clusters
+
+Define how pods communicate using network policies.
+
+Using VMware Tanzu Mission Control, you can create a network policy that defines how pods communicate with each other and other network endpoints, using preconfigured templates called recipes.
+By default, Tanzu Mission Control does not impose any such restriction, and allows you to manage network restrictions at the organizational level and at the workspace level.
+
+Tanzu Mission Control implements network policies using Kubernetes native network policies. Each namespace and workspace can be governed by a network policy, and these policies are inherited down through the organizational hierarchy.
```

Comment:
1. Q: Do we use comma before and in the terraform documentation? 
2. organization instead of organizational sounds much better -  `these policies are inherited down through the organization hierarchy.`

Reply:
1. Yes, we do use commas.
2. This snippet has been taken from the [official documentation](https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-39008830-8A6B-423E-B16E-EEC76B4DF9ED.html).